### DISCUSSION TITLE
How to get basic functionality working on node.js?

### DISCUSSION THREAD

**Discussion by ElixirMike** [handsontable/hyperformula#1140] [API Questions]
I've just added hyperformula to my node.js project and I'm trying to get basic functionality working. Here is my test method.
When I try to run this, I get this error: "HyperFormula.buildEmpty is not a function"

What am i missing?

```
const HyperFormula = require('hyperformula');

const hypertest = () => {

    const tableData = [
        ["Greg Black", 4.66, "=B1*1.3", "=AVERAGE(B1:C1)", "=SUM(B1:C1)"],
        ["Anne Carpenter", 5.25, "=$B$2*30%", "=AVERAGE(B2:C2)", "=SUM(B2:C2)"],
        ["Natalie Dem", 3.59, "=B3*2.7+2+1", "=AVERAGE(B3:C3)", "=SUM(B3:C3)"],
        ["John Sieg", 12.51, "=B4*(1.22+1)", "=AVERAGE(B4:C4)", "=SUM(B4:C4)"],
        [
          "Chris Aklips",
          7.63,
          "=B5*1.1*SUM(10,20)+1",
          "=AVERAGE(B5:C5)",
          "=SUM(B5:C5)"
        ]
      ];

      const hf = HyperFormula.buildEmpty({
        licenseKey: "gpl-v3"
      }); 

      // Add a new sheet and get its id.
const sheetName = hf.addSheet("main");
const sheetId = hf.getSheetId(sheetName);

// Fill the HyperFormula sheet with data.
hf.setCellContents(
  {
    row: 0,
    col: 0,
    sheet: sheetId
  },
  tableData
);

// Add named expressions for the "TOTAL" row.
hf.addNamedExpression("Year_1", "=SUM(main!$B$1:main!$B$5)");
hf.addNamedExpression("Year_2", "=SUM(main!$C$1:main!$C$5)");

}

module.exports = hypertest
```

**Comment by ElixirMike** [comment]
nm, needed to reference HyperFormula object 

HyperFormula.HyperFormula.buildEmpty({
        licenseKey: "gpl-v3"
      });

**Comment by sequba** [comment]
@ElixirMike, thanks for reaching out to us.

In our documentation, you'll find a [working example of a Node script running HyperFormula](https://hyperformula.handsontable.com/guide/file-import.html). This guide focuses on reading formulas from a file and calculating them using HyperFormula, but you might as well receive the input data from HTTP, database etc.

If you had any further questions about running HyperFormula in a Node environment, feel free to ask ;)
